client/v3/client_test.go

```diff
@@ -266,6 +266,29 @@ func TestSyncFiltersMembers(t *testing.T) {
 	}
 }
 
+func TestMultiEndpoints(t *testing.T) {
+	cfg := Config{
+		Endpoints:            []string{"192.168.3.41:22379", "192.168.3.41:22479", "192.168.3.41:22579"},
+		RejectOldCluster:     true,
+		DialTimeout:          5 * time.Second,
+		DialKeepAliveTime:    5 * time.Second,
+		DialKeepAliveTimeout: 5 * time.Second,
+		AutoSyncInterval:     time.Minute,
+	}
+	c, err := NewClient(t, cfg)
```

Comment:
The client is not `defer` Closed... thus it leaks goroutines and causes the tests to fail. 

(The issue would be easier to catch/debug if all the tests had: 
`testutil.BeforeTest(t)` as a first line). Would you mind adding in a separate PR ?

Reply:
I have revised it.